persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/StargateConfigSnitch.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.stargate.db.dse.impl;
+
+import io.stargate.db.datastore.common.StargateSnitchProperties;
+import java.net.InetAddress;
+import org.apache.cassandra.exceptions.ConfigurationException;
+import org.apache.cassandra.locator.GossipingPropertyFileSnitch;
+import org.apache.cassandra.utils.FBUtilities;
+
+/**
+ * The Stargate extension of the {@link GossipingPropertyFileSnitch}.
+ *
+ * <p><i>Note that similar copy of this class exists in all persistence implementations, thus please
+ * apply changes to all of them if needed.</i>
+ */
+public class StargateConfigSnitch extends GossipingPropertyFileSnitch {
+
+  private final StargateSnitchProperties snitchProperties;
+
+  public StargateConfigSnitch() throws ConfigurationException {
+    this(new StargateSnitchProperties("DEFAULT_DC", "DEFAULT_RACK"));
```

Comment:
Should we keep the existing defaults of `dc-1` and `rack1`?

Reply:
These above are existing defaults, have a look at [StargateConfigSnitch.java](https://github.com/stargate/stargate/blob/master/persistence-common/src/main/java/io/stargate/db/datastore/common/StargateConfigSnitch.java) being used at the moment. 

I am fine with changing the defaults to anything better if you @dougwettlaufer have a proposal?

Reply:
Looks like they're set to the C* defaults [here](https://github.com/stargate/stargate/blob/master/stargate-starter/src/main/java/io/stargate/starter/Starter.java#L353-L354) so by the time we get to `StargateConfigSnitch` they should be set anyway.

Reply:
OK, I am merging, but we can follow-up on this if needed..